clients/ui/frontend/src/utilities/utils.ts

```diff
@@ -0,0 +1,17 @@
+export const asEnumMember = <T extends object>(
```

Comment:
I think this already existed in the codebase. We can either keep one or if you feel this should be in the main utils I'm up for it.